cvat/apps/engine/views.py

```diff
@@ -3156,3 +3182,192 @@ def _import_project_dataset(request, rq_id_template, rq_func, db_obj, format_nam
     serializer.is_valid(raise_exception=True)
 
     return Response(serializer.data, status=status.HTTP_202_ACCEPTED)
+
+@extend_schema(tags=['requests'])
+@extend_schema_view(
+    list=extend_schema(
+        summary='List RQ jobs',
+        parameters=[
+            OpenApiParameter(name='task_id', location=OpenApiParameter.QUERY, type=OpenApiTypes.INT),
+            OpenApiParameter(name='action', location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, enum=['create']),
```

Comment:
This list of filters looks incomplete, comparing with the supported targets and actions. What do you think about adding filters like `project_id`, `job_id`, and other actions and subjects (e.g. backup, ...)? Also, what do you think about adding `status`, and time filters, sorting?